selfdrive/loggerd/logging.cc

```diff
@@ -0,0 +1,21 @@
+#include "selfdrive/loggerd/logging.h"
+
+#include "selfdrive/common/util.h"
+
+// Wait for all encoders to reach the same frame id
+bool sync_encoders(LoggerdState *s, CameraType cam_type, uint32_t frame_id) {
```

Comment:
I like how the sync logic is nicely contained in this function, but can we do this without adding new files? having all of `logger`, `logging`, and `loggerd` for filenames is confusing.

Reply:
need to move functions out of loggerd.cc for unit testing.
how about rename loggerd.cc-> main.cc, logging.cc->loggerd.cc, logging.h->loggerd.h?

Reply:
sync_encoders needs to be moved out from loggerd.cc for unit test.
how about rename loggerd.cc->main.cc, logging.cc->loggerd.cc, logging.h->loggerd.h ? just like proclogd.